.travis.yml

```diff
@@ -0,0 +1,11 @@
+language: php
+php:
+  - 5.4
+env:
+  global:
+    secure: DP5nm2y1uts/niMSRp4YtG1M076B87ukT1cZhtxMwxpmBxDMW6nqPpdJuN29GC00fSkF7FiOFCQD0BnO0RX5Me5xK+Cya2Cbpx04dRpUkqu3e5MThSBxaXNqgonf2Go6ndmy3ZSxCu7SpE+PvVU6fGdLClTs1d5RSDdvs5EGsRs=
```

Comment:
no, that's encrypted and needs to be added to the travis file as per their instructions: http://couscous.io/docs/travis.html ... unless I totally butchered it 😬